inference/environments/AzureML-minimal-ubuntu20.04-py38-cuda11.6.2-gpu-inference/environment.yaml

```diff
@@ -0,0 +1,26 @@
+image:
+  name: azureml/curated/AzureML-minimal-ubuntu20.04-py38-cuda11.6.2-gpu-inference
```

Comment:
Is this image name already in the mcr repo? It will need to be added before it can be released. The actual name should look similar to this, though: https://github.com/Azure/azureml-assets/blob/main/training/general/src/environments/pytorch-1.11-ubuntu20.04-py38-cuda11-gpu/environment.yaml#L2